### PR TITLE
Migrate avalanche to the new reducer

### PR DESCRIPTION
**Pull request by zurgl**
Migrate avalanche to the new reducer

### CODE REVIEW

**Review by pyr0gan — APPROVED** [review]
Pathway works 100% on Gitpod and local machine ✅

File: .env.example [hunk 1]
```diff
@@ -1,18 +1,15 @@
 # Datahub API Keys
-DATAHUB_AVALANCHE_API_KEY=
 DATAHUB_CELO_API_KEY=
 DATAHUB_NEAR_API_KEY=
 DATAHUB_POLKADOT_API_KEY=
 DATAHUB_POLYGON_API_KEY=
 DATAHUB_SECRET_API_KEY=
 DATAHUB_SOLANA_API_KEY=
 DATAHUB_TEZOS_API_KEY=
+DATAHUB_AVALANCHE_API_KEY=
```
Comment (guillaumegaluz) [inline .env.example:9]: This was in alphabetical order. Any reason for changing it?

File: .env.example [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 
 # Avalanche
-DATAHUB_AVALANCHE_MAINNET_RPC_URL=avalanche--mainnet--rpc.datahub.figment.io
-DATAHUB_AVALANCHE_FUJI_RPC_URL=avalanche--fuji--rpc.datahub.figment.io
-AVALANCHE_NETWORK_ID=5
-AVALANCHE_NETWORK_NAME=fuji
+AVALANCHE_DATAHUB_URL=avalanche--fuji--rpc.datahub.figment.io
```
Comment (guillaumegaluz) [inline .env.example:12]: Should we say FUJI in the name?

File: components/protocols/avalanche/components/Account.tsx [hunk 3]
```diff
@@ -1,66 +1,49 @@
...
-  useGlobalState,
-  getCurrentStepIdForCurrentChain,
-} from 'context';
 import {PROTOCOL_INNER_STATES_ID} from 'types';
-import {getAvalancheInnerState} from '@figment-avalanche/lib';
+import {useGlobalState} from 'context';
 
 const {Text} = Typography;
 
+const FAUCET = 'https://faucet.avax-test.network/';
```
Comment (guillaumegaluz) [inline components/protocols/avalanche/components/Account.tsx:9]: I'd call this FAUCET_URL

File: components/protocols/avalanche/components/Transfer.tsx [hunk 4]
```diff
@@ -19,46 +16,46 @@ const tailLayout = {
   wrapperCol: {offset: 4, span: 20},
 };
 
-const recipient = 'X-fuji1j2zasjlkkvptegp6dpm222q6sn02k0rp9fj92d';
+const RECIPIENT = 'X-fuji1j2zasjlkkvptegp6dpm222q6sn02k0rp9fj92d';
+const CHAIN_LABEL = 'Avalanche';
```
Comment (guillaumegaluz) [inline components/protocols/avalanche/components/Transfer.tsx:20]: We should use the one from "constants.ts"

File: components/protocols/secret/components/Account.tsx [hunk 5]
```diff
@@ -6,6 +6,8 @@ import {useGlobalState} from 'context';
 
 const {Text} = Typography;
 
+const FAUCET = 'https://faucet.supernova.enigma.co/';
```
Comment (guillaumegaluz) [inline components/protocols/secret/components/Account.tsx:9]: Same as above

File: components/protocols/secret/components/Transfer.tsx [hunk 6]
```diff
@@ -18,6 +18,8 @@ const tailLayout = {
 
 const {Text} = Typography;
 
+const CHAIN_LABEL = 'Secret';
```
Comment (guillaumegaluz) [inline components/protocols/secret/components/Transfer.tsx:21]: Would use the one from "constants.ts"

**Review by guillaumegaluz — COMMENTED** [review]
A few minor comments but otherwise looks great

**Pull request merged**